.travis.yml

```diff
@@ -1,8 +1,10 @@
 language: node_js
 node_js:
   - node
-  - lts/*
+  - 10
```

Comment:
Why was this necessary?

Reply:
Because `lts/*` just picks the last LTS branch but there are Node.js 6, 8 and 10 as LTS branch so 6 and 8 are not tested with `lts/*`.

Reply:
Regarding this (6 is still supported until April 2019), should we also test on Node.js 6?

Reply:
I would leave it like this.